.github/workflows/build-push.yml

```
@@ -0,0 +1,63 @@
+        run: |
+          echo "Switching to Windows Docker Daemon"
+          $contextName = docker context ls | Select-String -Pattern "windows" | ForEach-Object { $_.ToString().Trim() -split '\s+' | Select-Object -First 1 }
+          docker context use $contextName
+          docker info
+
+      - name: Cache Docker images
+        uses: actions/cache@v4
+        with:
+          path: /var/lib/docker
+          key: ${{ runner.os }}-docker-${{ hashFiles('**/windows.Dockerfile' && '**/linux.Dockerfile') }}
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)